src/libraries/Common/src/System/LocalAppContextSwitches.Common.cs

```diff
@@ -56,6 +56,11 @@ private static bool GetSwitchDefaultValue(string switchName)
                 return true;
             }
 
+            if (switchName == "Switch.System.Diagnostics.StackTrace.ILOffsetToStackTrace")
+            {
+                return Environment.GetEnvironmentVariable("COMPlus_ILOffsetToStackTrace") == "1" ? true : false;
```

Comment:
If you feel confident that you or someone on your team/company will benefit from being able to set that env var I am fine to include it. If 'someone' is a hypothetical .NET developer I'd rather wait to add it until we can clearly identify someone who will benefit from it.

Reply:
Also if we do have it, I'd suggest prefixing it with "DOTNET" rather than "COMPlus". For example: https://github.com/dotnet/runtime/blob/79ae74f5ca5c8a6fe3a48935e85bd7374959c570/src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionSettings.cs#L17

We are moving away from the 'COMPlus' naming, it is an obscure bit of .NET history that many devs probably don't relate to.

Reply:
@stephentoub do you have any particular opinion about allowing env variables to twiddle app context flags?